portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DataSourcesList.testcase

```diff
@@ -38,9 +25,25 @@ definition {
 	@ignore = "true"
```

Comment:
Hi @ViniLopes20 , please remove the ignore after updating the script.

Reply:
I removed in this commit: https://github.com/alexcandidos/liferay-portal/pull/497/commits/de6ce80a639e9683adc6c9c9158dadb54930691a